test/e2e/network/endpointslice.go

```diff
@@ -562,3 +565,17 @@ func createServiceReportErr(cs clientset.Interface, ns string, service *v1.Servi
 	framework.ExpectNoError(err, "error deleting Service")
 	return svc
 }
+
+func logEndpointSlice(epSlice discoveryv1beta1.EndpointSlice) {
```

Comment:
would it be better to marshal to JSON and log the full content? if there's a diff, it might not be in the few fields you selected to log

Reply:
I'd initially thought that would just end up polluting the logs but I think it's entirely worthwhile here, updated to use that approach, thanks!